internal/testing/utils.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2020 Google LLC
```

Comment:
I'd discourage naming this "testing". That will conflict with `testing.T`

Reply:
renamed it to utils